package.json

```diff
@@ -29,7 +29,7 @@
     "expo-dev-client": "~4.0.28",
     "expo-font": "~12.0.9",
     "expo-linking": "~6.3.1",
-    "expo-splash-screen": "~0.27.5",
+    "expo-splash-screen": "~0.27.7",
```

Comment:
why did you update it

Reply:
I updated the splash screen for improvement and to keep up with the latest version.